qcodes/instrument/parameter.py

```diff
@@ -1364,6 +1364,13 @@ def get(self):
             else:
                 return state['value']
 
+    def get_timestamp(self):
+        """
+        Return the age of the last get_latest call.
```

Comment:
this docstring is confusing to me. Is it the age of "last `get_latest` call" or "last `get` call on the parameter"? According to the code of `get_raw` of `_BaseParameter`, it should be the latter.

Reply:
Yep, that's right, docstring amended :)